src/Emulator/Visuals/Visuals.js

```diff
@@ -1,11 +1,169 @@
-import React from 'react';
+import React, {useRef, useState, useEffect} from 'react';
 import './Visuals.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearchPlus, faSearchMinus, faExpand, faEyeSlash} from '@fortawesome/free-solid-svg-icons';
+import Draggable from 'react-draggable';
 
 const Visuals = () => {
+  const defaultX = 322;
+  const defaultY = 322;
+  const [zoom, setZoom] = useState(true);
+  const [hideImage, setHideImage] = useState(false);
+  const [toggleMax, setMax] = useState(false);
+  const imageRef = useRef();
+  const [containerHeight] = useState('50vh');
+  const [containerWidth] = useState('75vw');
+  const [containerDimensions, setContainerDimensions] = useState({height: 0, width: 0});
+  const [contentDimensions, setContentDimensions] = useState({height: 0, width: 0});
+  const [image, setImage] = useState('null');
+  const [, setSrc] = useState('');
+  const [, setDrags] = useState(0);
```

Comment:
`const [, setSrc] = useState('');` allows me to change the images by clicking the buttons to switch between 200, 300, and 400px images
`const [, setDrags] = useState(0);` is currently not useful, but I thought it might be useful in the future because it will allow you to control what happens when the image if first clicked, and when the image is released from dragging. I took a publicly licensed drag library from GitHub, but unfortunately, it was made to be used with class components so I had a rough time converting it to work with functional components and this was one of the interesting things I found that gave me more control of what happens when dragging.